exercises/ruby-1-b/books.txt

```diff
@@ -0,0 +1,10 @@
+Title;ISBN;Author;Published
+The Adventures of Tom Sawyer;9780191604928;Mark Twain;2007
+Republic;9780718198916;Plato;2012
+Programming Ruby: The Pragmatic Programmers' Guide;9780974514055;David Thomas;2004
+Pride and Prejudice by Jane Austen;9781986431484;Jane Austen;2018
+To Kill a Mockingbird;9780446310789;Harper Lee;1988
+Cosmicomics;9780330319089;Italo Calvino;1969
+The Lord of the Rings;9780544003415;J. R. R. Tolkien;2012
+Lord of the Flies;9780140283334;William Golding;1999
+1984: A Novel;9780451524935;George Orwell;2009
```

Comment:
We should probably avoid using external files, since they would not be able to see what's in the file.

How about providing the file as a string inside the stub of the solution file?

Reply:
Why would they not be able to see the contents if they are able to read it?  Doesn't `debug` give us that ability?

Reply:
I am testing `debug` as described in the first one available (shopping cart) and it doesn't seem to work yet... not seeing anything when I use it and run the tests.